src/v2/view-builder/views/IdentifierView.js

```diff
@@ -1,34 +1,30 @@
-import { View, loc, createButton } from 'okta';
+import { loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
 import BaseFooter from '../internals/BaseFooter';
+import signInWithIdps from './signin/SignInWithIdps';
+import signInWithDeviceOption from './signin/SignInWithDeviceOption';
 
 const Body = BaseForm.extend({
 
   title: loc('primaryauth.title'),
   save: loc('oform.next', 'login'),
   initialize () {
     BaseForm.prototype.initialize.apply(this, arguments);
-    if (this.options.appState.hasRemediationForm('launch-authenticator')) {
-      this.add(View.extend({
-        className: 'sign-in-with-device-option',
-        template: `
-          <div class="okta-verify-container"></div>
-          <div class="separation-line"><span>OR</span></div>
-        `,
-        initialize () {
-          const appState = this.options.appState;
-          this.add(createButton({
-            className: 'button-secondary',
-            title: 'Sign in using Okta Verify',
-            click () { 
-              appState.trigger('invokeAction', 'launch-authenticator');
-            }
-          }), '.okta-verify-container');
-        }
-      }), '.o-form-fieldset-container', false, true);
-    }
-  }
+    this.listenTo(this.options.appState, 'formRendered', () => {
```

Comment:
prefer to override `render` instead of trigger event.

Reply:
I did try putting the logic inside postRender. But my challenge was that I had to wait till `o-form-button-bar` was available.

Reply:
did you try to override `render` ?

Reply:
I did not, I will try that and see if it works

Reply:
did not work as expected by overriding render as `.o-form-button-bar` is not rendered to the dom. Its almost like we want the identify form to finish rendering, and then we attach these buttons to that same container that is holding the next button.

Reply:
here is what I meant:
```
const Body = BaseForm.extend({

  title: loc('primaryauth.title'),
  save: loc('oform.next', 'login'),
  render () {
    BaseForm.prototype.render.apply(this, arguments);
    //this.listenTo(this.options.appState, 'formRendered', () => {
    if (this.options.appState.hasRemediationForm('launch-authenticator')) {
      this.add(signInWithDeviceOption, '.o-form-fieldset-container', false, true);
    }
    //add idps
    if (this.options.appState.get('idps')) {
      this.add(signInWithIdps, '.o-form-button-bar');
    }
    if (!this.options.appState.hasRemediationForm('identify')) {
      this.$el.find('.button-primary').hide();
      this.$el.find('.separation-line').hide();
    }
    //});
  },
}
```
Looks like works me locally. What have you tried and wonder what did I miss?

Reply:
nvm, I missed calling BaseForm.prototype.render.apply(this, arguments) 🤦